nemo/collections/tts/models/two_stages.py

```diff
@@ -0,0 +1,225 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, Dict
+
+import librosa
+import torch
+from hydra.utils import instantiate
+from omegaconf import MISSING, DictConfig, OmegaConf
+
+from nemo.collections.tts.helpers.helpers import griffin_lim
+from nemo.collections.tts.models.base import LinVocoder, MelToSpec, Vocoder
+from nemo.core.classes.common import PretrainedModelInfo
+from nemo.core.neural_types.elements import (
+    AudioSignal,
+    LengthsType,
+    MelSpectrogramType,
+    NormalDistributionSamplesType,
+    SpectrogramType,
+    VoidType,
+)
+from nemo.core.neural_types.neural_type import NeuralType
+from nemo.utils import logging
+
+
+class OperationMode(Enum):
+    """Training or Inference (Evaluation) mode"""
+
+    training = 0
+    validation = 1
+    infer = 2
```

Comment:
Please don't redefine this. We can even move this enum to base.py and just import it in files that use it.

Reply:
So, where should it come from?

It is placed there because I based it on WaveGlow, which defined it locally. 
Should I move the definition to somewhere local (eg. tts/helpers)?

Reply:
You are correct. We should move it from modules/WaveGlow. Sure, let's move it to tts/helpers